spiffworkflow-backend/src/spiffworkflow_backend/models/kkv_data_store.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+
+from spiffworkflow_backend.models.db import SpiffworkflowBaseDBModel
+from spiffworkflow_backend.models.db import db
+
+
+@dataclass
+class KKVDataStoreModel(SpiffworkflowBaseDBModel):
+    __tablename__ = "kkv_data_store"
+
+    id: int = db.Column(db.Integer, primary_key=True)
+    top_level_key: str = db.Column(db.String(255), index=True)
+    secondary_key: str = db.Column(db.String(255), index=True)
+    value: dict = db.Column(db.JSON)
+    updated_at_in_seconds: int = db.Column(db.Integer)
```

Comment:
i think jason and i decided that these blah_in_seconds columns should always be set and the columns should therefore be `nullable=False`. it's configured wrong in almost every model, but we might as well start correctly here.